atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/TableReference.java

```diff
@@ -41,15 +45,15 @@ public static TableReference createWithEmptyNamespace(String tablename) {
     }
 
     public static TableReference createUnsafe(String fullTableName) {
-        return fullTableName.indexOf('.') < 0 ? createWithEmptyNamespace(fullTableName) : createFromFullyQualifiedName(fullTableName);
-    }
-
-    public static boolean isFullyQualifiedName(String tableName) {
```

Comment:
Its name matched another method so checkstyle complained. It's a bit weird for this one because one is static and one is not.

In the general case, the rule makes it so overloaded methods are co-located.